packages/deployer/src/utils/files.ts

```diff
@@ -13,6 +18,28 @@ export function parseSecretsFile(secretsPath: string) {
   }
 }
 
+export function writeReceiptFile(
+  receiptFilename: string,
+  mnemonic: string,
+  config: Config,
+  commandOutput: TerraformAirnodeOutput
+) {
+  const airnodeId = deriveAirnodeId(mnemonic);
+  const receipt = {
+    airnodeId,
+    airnodeIdShort: shortenAirnodeId(airnodeId),
+    // Keeping `chains` and `nodeSettings` fields from the configuration and removing mnemonic
+    config: omit(pick(config, ['chains', 'nodeSettings']), 'nodeSettings.airnodeWalletMnemonic'),
```

Comment:
This should probably be done in a scalable way. We probably don't want any secret to be leaked out to `receipt.json`

Reply:
I have some solutions in mind for this, but it deserves discussion

Reply:
> Why do we _need_ the `config` in the receipt?

We need `stage`, `cloudProvider` and `region` in order to remove the deployment from `receipt.json`. Do you think it would be better to provide just those?

Also, I've just realized that currently the provider URL is also in the `receipt.json` and probably shouldn't be. Maybe we should rethink the `receipt.json` altogether.

What else (except those fields I mentioned) should be in the `receipt.json`? Maybe instead removing what is not needed we should just pick what is.

Reply:
> This should probably be done in a scalable way. We probably don't want any secret to be leaked out to `receipt.json`
> I have some solutions in mind for this, but it deserves discussion

I agree and I'm happy to hear your ideas but it looks like we may rework the `receipt.json` in general so I would wait till we decide what we actually want to do about it.

Reply:
I would say it's better to only include the fields we need like `stage`, `cloudProvider` and `region` rather than the entire config. It will be easy to add sensitive fields to the config and forget to remove them from the receipt.

Do we need the the provider URL in the receipt? 😄

Reply:
I moved the discussion to [slack](https://api3workspace.slack.com/archives/C02AYRX8D89/p1629448851051700), this is something that should be definitelly resolved in a separate PR.